### DISCUSSION TITLE
Add some examples about wavefront and PSF

### DISCUSSION THREAD

**Discussion by xjcclq** [mjhoptics/ray-optics#90] [Q&A]
Very useful and covers a wide range of areas in optics. DiffractionPSF is Fourier transform of wavefront. Could you add HuygensPSF and give some examples about wavefront and PSF? Thank you!

**Comment by mjhoptics** [comment]
There are a few Jupyter notebooks in my notebook repo that allow experimenting with wavefronts and PSF, i.e. to change the defocus and image centroid and see the effect on wavefront aberrations and PSF. See:

https://github.com/mjhoptics/ray-optics-notebooks/focus%20tilt%20notebook.ipynb
https://github.com/mjhoptics/ray-optics-notebooks/wavefront%20notebook.ipynb

Huygens PSF is a ways off.

Regards
Mike
